packages/docusaurus-mdx-loader/src/remark/toc/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -236,3 +236,23 @@ Some content here
 export const c = 1;
 "
 `;
+
+exports[`toc remark plugin works with imported markdown 1`] = `
+"import Partial from './_partial.md';
+
+export const toc = [
+	{
+		value: 'Thanos',
+		id: 'thanos',
+		level: 2
+	},
+	...toc0
```

Comment:
toc0 is not imported here, is this expected?

Reply:
Seems like we're going down the road of only modifying the AST and not the actual js, so this is expected.